common/solar_system_initializers/hfe_initializer.txt

```diff
@@ -120,49 +159,199 @@ hfe_initializer_egypt_main = {
 							show_notification = no
 						}
 						create_leader = {
-							name = Amonhirkhopshef
+							name = "Amonhirkhopshef Mutatistep"
 							species = event_target:hfe_egypt_species
 							class = ruler
-							skill = 5
+							skill = 10
 							leader_age_min = 250
 							leader_age_max = 650
+							immortal = yes
+							gender = male
 							traits = {
 								trait = leader_trait_ruler_chosen
 								trait = hfe_ruler_trait_transcendent_0
 							}
 						}
 						assign_leader = last_created_leader
 					}
-					save_global_event_target_as = hfe_egypt_country
```

Comment:
why

Reply:
I moved the Global Event Target elsewhere in the file. It's still there.